### ISSUE TITLE
[Suggestion]: Detailed suggestion on a "Hard Mode" config (Building on Issue #1638)

### ISSUE THREAD

**Issue by ghost** [nonamecrackers2/crackers-wither-storm-mod#1672]
### A greater challenge for Minecraft Pros

This suggestion is mainly to go into detail into the Hard Mode config mentioned in https://github.com/nonamecrackers2/crackers-wither-storm-mod/issues/1638, so the dev team can have a more clear view on what to do for a "Hard Mode". Free free to disregard any changes that seem "out of scope" for the mod.

Here's all of the changes to the mod for the Hard Mode config I propose:

### Wither Storm Changes

**- More damage upon being eaten by WS** The WS will deal more damage to players when they get eaten in Phase 4 and above.

**- Mobs and Players get Wither Sickness easier** Self-explanatory, All mobs and players will contract the Wither Sickness 1.5 times faster and the range of the WS giving the sickness is about 1.5 time the radius of influence.

**- Deadlier Projectiles** The skulls the WS will shoot out are faster and have greater blast radius.

**- Wither Storm moves faster** The WS will move faster when chasing players, about 1.2 times faster.

**- Deadlier Tractor Beam** The Tractor Beam will beam players quicker than usual.

### Withered Mob Changes

**- Stronger Mobs** All Withered Mobs will have more health, attack, and speed. about 1.2 times the default amount, and armor on the withered mobs can have stronger enchants like Protection IV, Thorns, etc. 

**- Withered Creeper has bigger blast radius** Self-explanatory, about 1.2 times the usual blast radius.

**- Withered Golem takes less damage from arrows** Self-explanatory, about half damage from arrows.  This forces the player to either risk fighting the golem at close range or deal with a more prolonged battle against the golem.

### Withered Symbiont Changes

**- Can cast Multiple Spells at once** In phase 1, the Wither Symbiont can cast 2 spells at once, making the symbiont much more  of a threat. In phase 2, it can cast 3 spells at once.

**- More Health** The Symbiont now has 100 health points.

### Command Block Changes

**- Harder Phases** This will go into detail on how each phase will be changed:

_Phase 1:_ The Tenacles are activated, a lot of mobs spawn (mainly Withered chickens, cows, mooshrooms, zombies, skeletons, spiders, and creepers), the CB is protected until all Tenacles are defeated.

_Phase 2:_ New Tenacles are made, stronger mobs spawn in (mainly Withered golems, pillagers, zombies, skeletons, spiders, and creepers), and a Withered Symbiont is spawned. The CB is protected until all Tenacles and Symbiont are defeated.
 
_Phase 3:_ Stronger mobs spawn in (mainly consisting of more Withered golems, phantoms, more creepers, and more pillagers), another Symbiont is spawned in, and the heads activate. The CB is protected until Symbiont and heads are defeated.

A new Achievement is achieved when being the WS in the Hard Mode config.
**Name:** Unwitherable
**Description:** Defeat the Wither Storm on Hard Mode.

### EDIT:

Some changes were made upon TaintedBlock's suggestions to keep the Hard Mode config not completely unfair:

**- All Mobs have a chance to give players Wither Sickness**  was removed due to Wither Sickness being a pain to remove.

**- Instant Death upon being eaten by WS** was changed to be more forgiving to the player. That said, https://github.com/nonamecrackers2/crackers-wither-storm-mod/issues/1637 was greenlit by crackers, so it's likely that config will be separate from the Hard Mode config.

All Phases were changed to be more fair and balanced, while Phase 1 had the Symbiont removed due to Phase 1 and 2 feeling very identical.

**Comment by TaintedBlockmObile** [comment]
I agree with most, but the wither sickness from sickened mobs, instant kill from head chomps, and bowels heads coming on twice with more tentacles doesn't seem too fair. For one, sickness is by itself annoying to get rid of and would easily get repetitive. Two, the heads instantly killing you basically makes going against the storm near impossible since no matter what armor you have, you'd die. I would say to ease on this the head should do more damage to people with armor on instead. Or do more on general. Three, bowels heads are extremely annoying along with tentacles and would make the bowels fight extremely unplayable in a lot of instances. And might just make players not want to fight in it. Maybe have the heads come back on stage 3 only? Tentacles coming back on phase 2 is fine I guess but should only spawn less instead of it being a basic respawn. But that is all for me.

**Comment by ghost** [comment]
> I agree with most, but the wither sickness from sickened mobs, instant kill from head chomps, and bowels heads coming on twice with more tentacles doesn't seem too fair. For one, sickness is by itself annoying to get rid of and would easily get repetitive. Two, the heads instantly killing you basically makes going against the storm near impossible since no matter what armor you have, you'd die. I would say to ease on this the head should do more damage to people with armor on instead. Or do more on general. Three, bowels heads are extremely annoying along with tentacles and would make the bowels fight extremely unplayable in a lot of instances. And might just make players not want to fight in it. Maybe have the heads come back on stage 3 only? Tentacles coming back on phase 2 is fine I guess but should only spawn less instead of it being a basic respawn. But that is all for me.

1. Fair enough, removed that suggestion.
2. Changed it, but https://github.com/nonamecrackers2/crackers-wither-storm-mod/issues/1637 was greenlit by crackers, so maybe it will be a separate config from the Hard Mode config.
3. All phases were changed to be more balanced, according to your suggestions.

**Comment by BluePineappleGuy** [comment]
This is exactly what i thought of when i made issue #1638.
The main idea was that these would be individual configs that you can toggle on and off, but the preset would activate/deactivate certain configs to enhance the difficulty of the storm. However, if its too hard for the player but the default is too easy, they can manually change the config to better suit them.

Also, an addition to the symbiont: it may or may not freeze when it summons the mobs, just because.

**Comment by BrunoGoldbergFerro** [comment]
Why a config? Wouldn't it be better if the changes were enabled if the difficulty was on Hard

**Comment by ghost** [comment]
> Why a config? Wouldn't it be better if the changes were enabled if the difficulty was on Hard

Gives more flexibility on how hard you want Minecraft as a whole to be. Much like how you can play Hard mode without Hardcore settings or change the difficulty in the settings menu on a world.

**Comment by Cod1ngGamer** [comment]
I like this idea. And to add on to it, maybe during the CB fight, chunks of blocks could fall from the roof like in https://github.com/nonamecrackers2/crackers-wither-storm-mod/issues/1395.
